### PR TITLE
[PORT] Notes now show the player's living playtime at the time of noting

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

- Does what it says on the tin

PORTS: 

https://github.com/tgstation/tgstation/pull/55800

https://github.com/tgstation/tgstation/pull/55893

## Why It's Good For The Game

Requested by headmin. 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

afUbeI58L8

Set up a DB, put in the 0h notes, then added this to the playtime table 

image

Then added the 2h notes.

</details>

## Changelog
:cl:
add: The notes menu now shows how many living hours the player had at the time the note was applied
/:cl:

### CODE REVIEW

**Comment by Crossedfall** [comment]
Frozen by https://github.com/BeeStation/BeeStation-Hornet/pull/7016

**Comment by llol111** [comment]
>ask to have something ported
>someone does it
feels good

Thank you for this, will make looking at notes alot more informational at a glance
since it will be possible to tell guy that plays one a month vs guy that playes all day evrydays

**Review by itsmeow — DISMISSED** [review]
Looks good

File: SQL/database_changelog.txt [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 0);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 1);
 
 In any query remember to add a prefix to the table names if you use one.
 
+Version 6.1, 29 December 2020, by Missfox, ported 20 August 2022 by ivanmixo
+Modified table `messages`, adding column `playtime` to show the user's playtime when the note was created.
+
+ALTER TABLE `messages` ADD `playtime` INT(11) NULL DEFAULT(NULL) AFTER `severity`
```
Comment (Crossedfall) [inline SQL/database_changelog.txt]:
```suggestion
ALTER TABLE `messages` ADD `playtime` INT(11) UNSIGNED NULL DEFAULT(NULL) AFTER `severity`
```

**Review by Crossedfall — APPROVED** [review]
Changes applied.

```sql
> INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (6, 1);
Query OK, 1 row affected (0.005 sec)

> ALTER TABLE `SS13_messages` ADD `playtime` INT(11) UNSIGNED NULL DEFAULT(NULL) AFTER `severity`;
Query OK, 0 rows affected (0.352 sec)
Records: 0  Duplicates: 0  Warnings: 0

> SHOW COLUMNS FROM SS13_messages;
+------------------+----------------------------------------------------------------+------+-----+---------+----------------+
| Field            | Type                                                           | Null | Key | Default | Extra          |
+------------------+----------------------------------------------------------------+------+-----+---------+----------------+
| id               | int(11)                                                        | NO   | PRI | NULL    | auto_increment |
| type             | enum('memo','message','message sent','note','watchlist entry') | NO   | MUL | NULL    |                |
| targetckey       | varchar(32)                                                    | NO   | MUL | NULL    |                |
| adminckey        | varchar(32)                                                    | NO   |     | NULL    |                |
| text             | varchar(2048)                                                  | NO   |     | NULL    |                |
| timestamp        | datetime                                                       | NO   |     | NULL    |                |
| expire_timestamp | datetime                                                       | YES  |     | NULL    |                |
| severity         | mediumtext                                                     | YES  |     | NULL    |                |
| playtime         | int(11) unsigned                                               | YES  |     | NULL    |                |
| server_name      | varchar(32)                                                    | YES  |     | NULL    |                |
| server_ip        | int(10) unsigned                                               | NO   |     | NULL    |                |
| server_port      | smallint(5) unsigned                                           | NO   |     | NULL    |                |
| round_id         | int(11) unsigned                                               | NO   |     | NULL    |                |
| secret           | tinyint(1) unsigned                                            | NO   |     | NULL    |                |
| lasteditor       | varchar(32)                                                    | YES  |     | NULL    |                |
| edits            | mediumtext                                                     | YES  |     | NULL    |                |
| deleted          | tinyint(1) unsigned                                            | NO   |     | 0       |                |
+------------------+----------------------------------------------------------------+------+-----+---------+----------------+
17 rows in set (0.001 sec)
```

**Pull request merged**